common/autoconf/configure

```diff
@@ -123,7 +123,15 @@ fi
 ###
 # Add default OpenJ9 closed extensions
 full_script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-conf_processed_arguments=("--with-custom-make-dir=$full_script_dir/../../closed/make" "--with-add-source-root=$full_script_dir/../../closed/adds")
+OPENJDK_TAG=`git -C $full_script_dir/../.. describe --abbrev=0 --tags`
```

Comment:
Its a pre-existing problem in the place where this was copied from, but this in should have `--match "jdk8*" ${OPENJDK_SHA}` and OPENJDK_SHA would need to be defined as well.

Reply:
It could be fixed in the custom-hook.m4 as well, since this file is already being changed.